src/main/java/ch/admin/bag/covidcertificate/authorization/AuthorizationConfig.java

```diff
@@ -7,15 +7,37 @@
 import org.springframework.context.annotation.Configuration;
 
 
+/**
+ * Holder object for all named services that need a service configuration using ServiceData.
+ */
 @Data
 @Configuration
 @ComponentScan
 @ConfigurationProperties(prefix = "services")
 public class AuthorizationConfig {
 
+    /**
+     * The service called Web UI that uses the management service to generate, revoke, covid certificates.
+     */
     private ServiceData webUi;
+
+    /**
+     * The API Gateway service that uses the management service to generate, revoke covid certificates.
+     */
     private ServiceData apiGateway;
+
+    /**
+     * The management service as central service to generate, revoke, print, sign covid certificates.
```

Comment:
besser: "...core service to generate, revoke, print and sign covid certificates"